tests/conftest.py

```diff
@@ -465,6 +465,12 @@ def create_client(self, urgent=False):
             if urgent:
                 w.set_wm_hints(flags=Xutil.UrgencyHint)
 
+            if pid is not None:
```

Comment:
You can just say `if pid` here (...is what The-Compiler would say :wink: )

Reply:
Is this really equivalent for `pid=0`?

Reply:
Oh, you're totally right. Disregard that.

Reply:
Being more explict: assume that I call `create_client` with `pid=0`, then `pid is not None` is `True` and we indeed want the x-property to be set. However `bool(pid)` evaluates to `False` and this would cause the xproperty to be left unset (which we don't want).